src/ParcelRegistry.Projections.Legacy/Migrations/20221123144056_AddV2Tables_P2.cs

```diff
@@ -20,7 +20,7 @@ protected override void Up(MigrationBuilder migrationBuilder)
             WITH SCHEMABINDING
             AS
             SELECT COUNT_BIG(*) as Count
-            FROM [{Infrastructure.Schema.Legacy}].[{ParcelDetailV2.ParcelDetailV2Configuration.TableName}]
+            FROM [{Infrastructure.Schema.Legacy}].[ParcelDetailsV2]
```

Comment:
Shouldn't it be ParcelDetails? 

```
    public class ParcelDetailConfiguration : IEntityTypeConfiguration<ParcelDetail>
    {
        internal const string TableName = "ParcelDetails";
```

Reply:
Not at this point. It's still ParcelDetailsV2